imports/api/posts/server/publications.js

```diff
@@ -1,7 +1,8 @@
 import { Meteor } from 'meteor/meteor';
 import { Posts } from '../posts_collection.js'
+import { pagination } from 'meteor/kurounin:pagination';
 
-Meteor.publish('posts', function (options) {
+Meteor.publish('posts0', function (options) {
```

Comment:
I'm betting there's an interesting reason these publication names had to be appended with a number. If they can't be renamed to something intuitive, maybe add comments on why they got the funny names?